actions/npm-check/npm_check.js

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env node
+
+// trunk-ignore-all(eslint)
+const npmCheck = require("npm-check");
+const YAML = require("yaml");
+
+// TODO(lauri): Find more stable icon path
+const ICON_PATH = "https://avatars.githubusercontent.com/u/6078720";
```

Comment:
Since the ICON path should be able to construct from a local file path - I always wanted the plugin repo to be able to source custom icons inilne. So if you took this SVG and put it inside yarn-check you could serve it up. 

The notification returned would need to learn how to build a fully qualified path from thedisk 

So in theory you'd convert relative to absolute file paths on disk - and avoid having to hit the network for the icon.

Reply:
Agree. This should be doable if you pass in ${plugin} or ${cwd} as an input to this script, right?

Reply:
Had to patch the extension to support this but yes, changed it to be local icons.